websites/docs-smoke-test/src/release-notes/bugfix.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Release notes not reappearing when changed
+date: 2020-03-16
+type: Resolved Issue
```

Comment:
Wondering if we should provide here only "identifiers". The actual "labels" are then mapped based on the identifier.
The reason here is to avoid possible typos and wrong letter casing.

For example: `new`, `resolved`, etc.

Reply:
good idea for the types since they are intended to be finite and predefined, thanks !

Reply:
how about `feature` , `fix` and `enhancement`??